native/Avalonia.Native/src/OSX/window.mm

```diff
@@ -1360,6 +1371,16 @@ - (void)windowDidDeminiaturize:(NSNotification *)notification
     }
 }
 
+- (void)windowDidResize:(NSNotification *)notification
+{
+    auto parent = dynamic_cast<IWindowStateChanged*>(_parent.operator->());
```

Comment:
Why is this needed?

Reply:
removed cast, and interface method. not required.